samples/first-time-setup.yml

```
@@ -0,0 +1,28 @@
+name: "Version uptick automation"
```

Wrong title for sample workflow

```suggestion
name: "Repository First Time Setup automation"
```